.github/workflows/code-coverage.yml

```diff
@@ -39,6 +39,7 @@ jobs:
     - name: Build HELICS
       run: |
         unset VCPKG_ROOT
+        git config --global --add safe.directory /__w/HELICS/HELICS
```

Comment:
I think this should either use the `$GITHUB_WORKSPACE` environment variable or `${{ github.workspace }}` workflow variable.

Reply:
done